syncer/syncer.go

```diff
@@ -202,7 +202,7 @@ type Syncer struct {
 
 	tsOffset            atomic.Int64             // time offset between upstream and syncer, DM's timestamp - MySQL's timestamp
 	secondsBehindMaster atomic.Int64             // current task delay second behind upstream
-	workerLagMap        map[string]*atomic.Int64 // worker's sync lag key:queueBucketName val: lag
+	workerLagMap        map[string]*atomic.Int64 // worker's sync lag key:WorkerLagKey val: lag
```

Comment:
Shall we just use `[]*atomic.Int64` here? the workerId should be the slice index?

Reply:
I think the string key is better for reading as there are skipped jobs and ddl jobs that also need to be calculated separately.

Reply:
ok, lgtm. I'm not sure if we we pack worker log related logic into a separated struct then since ddl and skip jog are special logic, we can move them out of the map and convert to map into a slice.

Reply:
ok, maybe anothr PR i will do this
BTW could you tell me why we should use slice rather than map ?

Reply:
https://www.google.com/search?q=use+slice+rather+than+map&oq=use+slice+rather+than+map&aqs=chrome..69i57j69i64.3091j0j7&sourceid=chrome&ie=UTF-8